scripts/detect_duplicate_features.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+import yaml
+
+
+def findall_features(features):
+    feature_list = []
+    for feature in features:
+        if "and" in feature:
+            and_list = findall_features(feature["and"])
+            for x in and_list:
+                feature_list.append(x)
+        elif "or" in feature:
+            or_list = findall_features(feature["or"])
+            for y in or_list:
+                feature_list.append(y)
+        else:
+            feature_list.append(feature)
+
+    return feature_list
+
+
+def find_overlapping_rules(new_rule_path, rules_path):
+    if not new_rule_path.endswith(".yml"):
+        return "ERROR ! New rule path file name incorrect"
+
+    count = 0
+
+    with open(new_rule_path, "r") as f:
+        new_rule = yaml.safe_load(f)
```

Comment:
have you looked into `capa.rules.Rule.from_yaml`?

Reply:
Thanks for the feedback.
Will look into it.

Reply:
did this not turn out fruitful?

Reply:
Sorry, missed it. Working on it.
Code needs a few changes since I some structural nodes.
Will update soon.